File: src/main/java/com/core/base/corebase/controller/review/dto/QuestionRes.java
// Copyright 2025 Kakao Healthcare Corp. All Rights Reserved.

package com.core.base.corebase.controller.review.dto;

import com.core.base.corebase.domain.review.code.QuestionType;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;
import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class QuestionRes {
    private UUID id;
    private String question;
    private QuestionType type;
    private List<ChoiceRes> choices;
    private Integer limit;
    private Integer orderNum;
    private Boolean useScore;
    private Boolean useMultiSelect;
}

File: src/main/kotlin/com/core/base/corebase/controller/review/dto/QuestionRes.kt
package com.core.base.corebase.controller.review.dto

import com.core.base.corebase.domain.review.code.QuestionType
import java.util.*

class QuestionRes (
    val id : UUID,
    val question: String,
    val type: QuestionType,
    val choices: List<ChoiceRes>?,
    val limit: Int?,
    val order: Int,
    var useScore: Boolean,
    var useMultiSelect: Boolean
) 